src/api/Base/index.js

```diff
@@ -7,7 +7,7 @@ const API = axios.create({
   // baseURL: `${process.env.REACT_APP_SERVER_API}/api/v1`,
   baseURL: 'http://localhost:82/api/v1',
   headers: {
-    'Content-Type': 'application/json',
+    'Content-Type': 'application/json; multipart/form-data',
```

Comment:
please remove the multipart/form-data. 
```suggestion
    'Content-Type': 'application/json',
```